### PR TITLE
Bugfix: Fix minor gauntlet ocr2:set_billing bug, add validation

### PR DESCRIPTION
**Pull request by reductionista**
gauntlet was erroring out if recommendedGasPriceUluna=1.0, even
though it's supposed to be a Decimal field.  Removed conversion
to BN which can't parse decimals, added validation function
instead.

### CODE REVIEW

**Review by archseer — DISMISSED** [review]
LGTM 👍🏻 

So the value is a string in RDD right?

File: packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts
Comment (RodrigoAD) [inline packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts:44]: Let's use `const` every time we can. Also, `gas_price` should be camelCase

File: packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts
Comment (RodrigoAD) [inline packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts:71]: Let's avoid nesting if possible for readability. This could be:
```suggestion
 try {
    observationPayment = BN(input.observationPaymentGjuels)
    transmissionPayment = BN(input.transmissionPaymentGjuels) // parse as integers
  } catch {
    throw new Error(
      `observationPaymentGjuels=${input.observationPaymentGjuels} and ` +
        `transmissionPaymentGjuels=${input.transmissionPaymentGjuels} must both be integers`,
    )
  }
  if (observationPayment.isNeg() || transmissionPayment.isNeg()) {
    throw new Error(
      `observationPaymentGjuels=${input.observationPaymentGjuels} and ` +
        `transmissionPaymentGjuels=${input.transmissionPaymentGjuels} cannot be negative`,
    )
  }
```

**Comment by reductionista** [comment]
> LGTM 👍🏻
> 
> So the value is a string in RDD right?

Yes, presently they are all set to "1" in the RDD... changed them to "1.0" on a branch to test it, but either will work.

**Comment by reductionista** [comment]
Comments have been addressed, waiting for Approve to merge

**Review by krebernisak — COMMENTED** [review]
Please run Gautlet linter locally before merging, looks like that check failed.

Rebase should also fix the e2e test failure.

**Comment by reductionista** [comment]
yarn format ran successfully, ready to be merged

**Review by archseer — APPROVED** [review]

**Pull request merged**
